src/pages/Login.js

```diff
@@ -1,9 +1,99 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { saveUser } from '../actions';
+
+class Login extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      email: '',
+      password: '',
+      disable: true,
+    };
+
+    this.HandleChange = this.HandleChange.bind(this);
+    this.FormValidate = this.FormValidate.bind(this);
+    this.OnSubmitUser = this.OnSubmitUser.bind(this);
+  }
+
+  HandleChange({ target }) {
+    const { name, value } = target;
+    this.setState({
+      [name]: value,
+    });
+    this.FormValidate();
+  }
+
+  OnSubmitUser() {
+    const { email } = this.state;
+    const { SetUserStore } = this.props;
+    SetUserStore(email);
+  }
+
+  FormValidate() {
+    const { email, password } = this.state;
+    const lengthMin = 5;
+    const validEmail = /^[a-z0-9_.-]+@[a-z]+\.[a-z]{2,3}(?:\.[a-z]{2})?$/; // Regex criada por Rodrigo Merlone e disponibilizada no slack;
+    if (validEmail.test(email) && password.length >= lengthMin) {
+      this.setState({ disable: false });
+    } else {
+      this.setState({ disable: true });
+    }
+  }
 
-class Login extends React.Component {
   render() {
-    return <div>Login</div>;
+    const { email, password, disable } = this.state;
+    return (
+      <div>
+        TrybeLogin
+        <form>
+          <label htmlFor="email" data-testid="email-input">
+            email:
+            <input
+              type="email"
```

Comment:
Acredito que o `data-testid` seja no `input` do e-mail, assim como colocou no `input` da senha. O que acha de alterar esse detalhe? :smile: 

```suggestion
          <label htmlFor="email">
            email:
            <input
              data-testid="email-input"
              type="email"
```